test/integration/validate.test.js

```diff
@@ -38,39 +18,44 @@ describe('validate', () => {
     };
     const result = validate(request, request);
 
-    isValid(result);
+    it('marks pair as valid', () => {
+      expect(result).to.be.valid;
+    });
 
     it('contains all validatable keys', () => {
-      assert.hasAllKeys(result.fields, ['method', 'headers', 'body']);
+      expect(result.fields).to.have.all.keys(['method', 'headers', 'body']);
     });
 
     describe('method', () => {
-      isValid(result.fields.method);
-      validator(result.fields.method, null);
-      expectedType(result.fields.method, 'text/vnd.apiary.method');
-      realType(result.fields.method, 'text/vnd.apiary.method');
+      expect(result.fields.method).to.be.valid;
+      expect(result.fields.method).to.have.validator(null);
+      expect(result.fields.method).to.have.expectedType(
+        'text/vnd.apiary.method'
+      );
+      expect(result.fields.method).to.have.realType('text/vnd.apiary.method');
+
       noErrors(result.fields.method);
```

Comment:
`noErrors` is intentionally going to be a next step or was it forgotten?

Reply:
The next step. I haven't yet decided what would be a comfortable interface to assert Gavel's validation errors, so I left errors assertions behind.